e2e/cypress/integration/plugins/upgrade_spec.js

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @system_console @plugin
+
+/**
+ * Note : This test requires two demo plugin tar files under fixtures folder.
+ * Download version 0.1.0 from :
+ * https://github.com/mattermost/mattermost-plugin-demo/releases/download/v0.1.0/com.mattermost.demo-plugin-0.1.0.tar.gz
+ * Copy to : ./e2e/cypress/fixtures/com.mattermost.demo-plugin-0.1.0.tar.gz
+ *
+ * Download version 0.2.0 from :
+ * https://github.com/mattermost/mattermost-plugin-demo/releases/download/v0.2.0/com.mattermost.demo-plugin-0.2.0.tar.gz
+ * Copy to : ./e2e/cypress/fixtures/com.mattermost.demo-plugin-0.2.0.tar.gz
+ */
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Plugin remains enabled when upgraded', () => {
+    const pluginIdDemo = 'com.mattermost.demo-plugin';
+
+    before(() => {
+        // # Update config
+        cy.apiUpdateConfig({
+            PluginSettings: {
+                Enable: true,
+                RequirePluginSignature: false,
+            },
+        });
+
+        // # Initialize setup and visit town-square
+        cy.apiInitSetup().then(({team}) => {
+            cy.visit(`/${team.name}/channels/town-square`);
+
+            // #If Demo plugin is already enabled , unInstall it
+            cy.apiRemovePluginById(pluginIdDemo);
+            cy.visit('/admin_console/plugins/plugin_management');
+            cy.get('.admin-console__header', {timeout: TIMEOUTS.ONE_MIN}).should('be.visible').and('have.text', 'Plugin Management');
+        });
+    });
+
+    it('MT40-Plugin remains enabled when upgraded', () => {
+    // * upload Demo plugin from the browser
```

Comment:
nt: please capitalize comment
```suggestion
    // * Upload Demo plugin from the browser
```

Reply:
done.